docs/merge_points/merge-action-groups.md

```diff
@@ -0,0 +1,74 @@
+# Merge action groups
+
+An action group is a set of individual actions working together as a group.
+These action groups can be shared between tests and they also be modified to your needs.
+
+In this example we add a `<click>` command to check the checkbox that our extension adds to the simple product creation form.
+
+## Starting test
+
```

Comment:
```suggestion
<!-- raw -->
```
the text below contains `{{ }}` symbols that have to be escaped for Liquid parser